master compilation failure due to undef'd fontbase

I happened to download master because I needed a cmake-enabled glvis.

I tried to build on OSX 10.11.6. I had to turn off PNG and FREETYPE due to...

```
ld: warning: ld: warning: ignoring file /sw/lib/libpng.dylib, file was built for i386 which is not the architecture being linked (x86_64): /sw/lib/libpng.dylibignoring file /sw/lib/libfreetype.dylib, file was built for i386 which is not the architecture being linked (x86_64): /sw/lib/libfreetype.dylib
```
But, then I ran into this error...
```
/Users/miller86/fastmath/atpesc17_examples/mfem/glvis/lib/vssolution.cpp:1020:16: error: use of undeclared identifier 'fontbase'
```
Adding ``extern GLuint fontbase;`` declaration to to vssolution.cpp fixes

--- a/lib/aux_vis.hpp
+++ b/lib/aux_vis.hpp
@@ -21,6 +21,7 @@
 
 #include "openglvis.hpp"
 
+extern GLuint fontbase;
 extern float MatAlpha;
 extern float MatAlphaCenter;
 

--- a/lib/vsdata.cpp
+++ b/lib/vsdata.cpp
@@ -24,8 +24,6 @@ using namespace std;
 
 #include "gl2ps.h"
 
-extern GLuint fontbase;
-
 const char *strings_off_on[] = { "off", "on" };
 
 void VisualizationSceneScalarData::FixValueRange()

